.eslintrc.js

```diff
@@ -45,7 +45,6 @@ module.exports = {
 
     // TODO reactivate all the following rules
 
-    'func-names': 'off',
```

Comment:
Can you put this file `.eslintrc.js ` in separate commit? 
To facilitate cherry-pick and rebase.